fili-security/src/test/groovy/com/yahoo/bard/webservice/web/security/RoleBasedRoutingRequestMapperSpec.groovy

```diff
@@ -0,0 +1,67 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web.security
+
+import com.yahoo.bard.webservice.config.SystemConfigProvider
+import com.yahoo.bard.webservice.data.config.ResourceDictionaries
+import com.yahoo.bard.webservice.web.ApiRequest
+import com.yahoo.bard.webservice.web.RequestMapper
+
+import spock.lang.Specification
+import spock.lang.Unroll
+
+import javax.ws.rs.container.ContainerRequestContext
+import javax.ws.rs.core.SecurityContext
+
+class RoleBasedRoutingRequestMapperSpec extends Specification {
+
+    LinkedHashMap<String, RequestMapper<ApiRequest>> prioritizedRoleBasedMappers;
+    RequestMapper<ApiRequest> mapperA = Mock(RequestMapper)
+    RequestMapper<ApiRequest> mapperB = Mock(RequestMapper)
+    RequestMapper<ApiRequest> mapperC = Mock(RequestMapper)
```

Comment:
It's used down in line 29 building the mapper, and in the next mapper test at the bottom.